packages/sd-jwt/src/action-handler.ts

```diff
@@ -0,0 +1,227 @@
+import { schema } from './index'
+import { Jwt, SDJwt } from '@sd-jwt/core'
+import { SDJwtVcInstance, SdJwtVcPayload } from '@sd-jwt/sd-jwt-vc'
+import { Signer, Verifier, KbVerifier, JwtPayload, DisclosureFrame, PresentationFrame } from '@sd-jwt/types'
+import { IAgentPlugin } from '@veramo/core'
+import {
+  SdJWTImplementation,
+  ICreateSdJwtVcArgs,
+  ICreateSdJwtVcResult,
+  ICreateSdJwtVcPresentationArgs,
+  ICreateSdJwtVcPresentationResult,
+  IRequiredContext,
+  ISDJwtPlugin,
+  IVerifySdJwtVcArgs,
+  IVerifySdJwtVcResult,
+  IVerifySdJwtVcPresentationArgs,
+  IVerifySdJwtVcPresentationResult,
+  Claims,
+} from './types.js'
+import { mapIdentifierKeysToDocWithJwkSupport } from '@sphereon/ssi-sdk-ext.did-utils'
+import { encodeJoseBlob } from '@veramo/utils'
+
+/**
+ * @beta
+ * SD-JWT plugin for Veramo
+ */
+export class SDJwtPlugin implements IAgentPlugin {
+  readonly schema = schema.ISDJwtPlugin
+
+  constructor(private algorithms: SdJWTImplementation) {}
+
+  // map the methods your plugin is declaring to their implementation
+  readonly methods: ISDJwtPlugin = {
+    createSdJwtVc: this.createSdJwtVc.bind(this),
+    createSdJwtVcPresentation: this.createSdJwtVcPresentation.bind(this),
```

Comment:
I know this is from the original plugin, but suggest to remove Vc from the Presentation function names. It is unneeded and confusing

Reply:
renamed